python/api/application.py

```diff
@@ -56,9 +57,15 @@ def default(self, obj):
 #Allow cross-origin requests. TODO should eventually lock down the permissions on this a bit more strictly, though only allowing GET requests is a good start.
 CORS(application, resources={r"/api/*": {"origins": "*"}}, methods=['GET'])
 
+#Allow us to test locally if desired
+if 'docker' in sys.argv:
+    database_choice = 'docker_database'
+else:
+    database_choice = 'remote_database'
+
 with open('secrets.json') as f:
```

Comment:
I ran into an issue with this as a relative path - I think this needs to be turned into a relative path in reference to the application.py file path. Unless I'm missing something, currently you'd have to be in `python/housinginsights` path and then run `python ../api/application.py docker` for it run correctly.